examples/react/start-workos/.env.example

```diff
@@ -0,0 +1,5 @@
+# The following environment variables are required to run the application
```

Comment:
Please add a general Node `.gitignore` and have `.env` added in, whilst still leaving `.env.example` as visible.